app/controllers/microservices_engine/v1/data_controller.rb

```diff
@@ -42,11 +42,11 @@ def update
         data = params['content']
         if data
```

Comment:
The idea is that data is not required but can hold an empty hash if no updates occurred.